pyproject.toml

```diff
@@ -9,13 +9,15 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.10"
 langchain = "^0.0.215"
-python-dotenv = "^1.0.0"
+python-dotenv = "^0.21.0"
 openai = "^0.27.8"
 autopack-tools = "^0.2.0"
 psutil = "^5.9.5"
-agent-protocol = "^0.2.0"
+agent-protocol = "^0.2.2"
 helicone = "^1.0.6"
-agbenchmark = "^0.0.3"
+
+tenacity = "^8.2.2"
+agent-protocol-client = "^0.2.2"
```

Comment:
this won't be needed

Reply:
At what point will this not be needed?

Reply:
After you simplify to the API benchmark format